### PR TITLE
tooling: remove tools/Makefile

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

We use docker for all protobuf related items. This makes it unnecessary to provide a way to download tooling. 

ref #6103

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -429,7 +412,7 @@ most probably (99.9%)*.
 [Jepsen](http://jepsen.io/) tests are used to verify the
 [linearizability](https://jepsen.io/consistency/models/linearizable) property
 of the Tendermint consensus. They are located in a separate repository
--> https://github.com/tendermint/jepsen. Please refer to it's README for more
+-> <https://github.com/tendermint/jepsen>. Please refer to it's README for more
```
Comment (tessr) [inline CONTRIBUTING.md:415]:
```suggestion
-> <https://github.com/tendermint/jepsen>. Please refer to its README for more
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -108,24 +108,7 @@ We use [Protocol Buffers](https://developers.google.com/protocol-buffers) along
...
-To install `gogoproto`, do the following:
-
-```sh
-go get github.com/gogo/protobuf/gogoproto
-cd $GOPATH/pkg/mod/github.com/gogo/protobuf@v1.3.1 # or wherever go get installs things
-make install
-```
-
-You should now be able to run `make proto-gen` from inside the root Tendermint directory to generate new files from proto files.
+We use [Docker](https://www.docker.com/) to generate the protobuf stubs. To generate the stubs yourself, make sure docker is running then run `make proto-gen-docker`.
```
Comment (marbar3778) [inline CONTRIBUTING.md:111]:
```suggestion
We use [Docker](https://www.docker.com/) to generate the protobuf stubs. To generate the stubs yourself, make sure docker is running then run `make proto-gen`.
```

**Review by tessr — APPROVED** [review]
LGTM! Please also backport to v0.34.x

**Comment by marbar3778** [comment]
@Mergifyio backport v0.34.x

**Pull request merged**